test/test_sparse.py

```diff
@@ -744,6 +744,8 @@ def test_sparse_variable_methods(self):
         to_test_mixed = {
             'addmm': lambda sp, de: de.addmm(sp, de),
             'addmm_': lambda sp, de: de.addmm(sp, de),
+            'mm': lambda sp, de: torch.mm(sp, de),
+            'mm_out': lambda sp, de: torch.mm(sp, de, out=de),
```

Comment:
do these things check the gradients work?

Reply:
Nope. The gradients won't work because we don't have `dense * sparse` multiplication, only `sparse * dense`